src/burnchains/burnchain.rs

```diff
@@ -892,7 +892,7 @@ impl Burnchain {
 
     /// Determine if there has been a chain reorg, given our current canonical burnchain tip.
     /// Return the new chain tip
```

Comment:
What is the new return type? You added a boolean, right? What is it doing?

Reply:
Addressed with latest